examples/torch/ppo_metaworld_mt10.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+"""This is an example to train PPO on MT10 environment."""
+# pylint: disable=no-value-for-parameter
+import click
+from metaworld.benchmarks import MT10
+import torch
+
+from garage import wrap_experiment
+from garage.envs import MultiEnvWrapper
+from garage.envs.multi_env_wrapper import round_robin_strategy
+from garage.experiment import LocalRunner
+from garage.experiment.deterministic import set_seed
+from garage.np.baselines import LinearFeatureBaseline
+from garage.torch.algos import PPO
+from garage.torch.policies import GaussianMLPPolicy
+
+
+@click.command()
+@click.option('--seed', default=1)
+@click.option('--epochs', default=500)
+@click.option('--batch_size', default=1024)
+@wrap_experiment(snapshot_mode='all')
+def torch_ppo_mt10(ctxt, seed, epochs, batch_size):
+    """Set up environment and algorithm and run the task.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+        epochs (int): Number of training epochs.
+        batch_size (int): Number of environment steps in one batch.
+
+    """
+    set_seed(seed)
+    tasks = MT10.get_train_tasks().all_task_names
+    envs = []
+    for task in tasks:
+        envs.append(MT10.from_task(task))
+    env = MultiEnvWrapper(envs,
+                          sample_strategy=round_robin_strategy,
+                          metaworld_mt=True)
```

Comment:
You should probably wait for Avnish's change, which is going to move / change the `metaworld_mt` parameter.

Reply:
Actually, Keren go to this change first, so please update to use the change he made [here](https://github.com/rlworkgroup/garage/commit/8d8051d830f2448b0777e475ea6baad72f23b6dc#diff-1cf058b9e44a937f666bf0e25e9323bdR57).